src/components/RemoteConfig/index.tsx

```diff
@@ -41,21 +41,34 @@ function RemoteConfig() {
     useTemplate();
 
   const [searchText, setSearchText] = useState('');
+  const [edited, setEdited] = useState(false);
   const [paramBeingEdited, editParam] = useState<string | undefined>(undefined);
   const [editTemplate, setEditTemplate] = useState(
     JSON.parse(JSON.stringify(template))
   );
 
-  const editing = isEqual(template, editTemplate);
+  const editing = isEqual(template, editTemplate) && !edited;
```

Comment:
This seems backwards to me - should it be:

```suggestion
  const editing = !isEqual(template, editTemplate) && edited;
```